manager/integration/tests/test_ha.py

```diff
@@ -1429,8 +1431,7 @@ def test_volume_reattach_after_engine_sigkill(
                                  data_path2, DATA_SIZE_IN_MB_1)
     md5sum2 = get_pod_data_md5sum(core_api, pod_name, data_path2)
 
-    res_md5sum2 = get_pod_data_md5sum(core_api, pod_name, data_path2)
-    assert md5sum2 == res_md5sum2
+    assert md5sum2 is not None
```

Comment:
? How can this get passed before?

Reply:
I saw this code snippet, I think asserting ```md5sum2 == res_md5sum2``` was not making sense, both are same computed value so I changed it. This is not related to the failure of the test.

Reply:
It's better to write the specific data then read and verify it. e.g., using `echo` to write some strings and then using `cat` to read the file and verify the strings.

Reply:
Updated this with write and reading a string using ```echo``` and ```cat```.